sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -78,22 +79,24 @@ def _from_generated(cls, gen):
 class ArtifactManifestProperties(object):
     """Represents properties of a registry artifact
 
-    :ivar str cpu_architecture: CPU Architecture of an artifact
+    :ivar architecture: CPU Architecture of an artifact
+    :vartype architecture: ~azure.containerregistry.ArtifactArchitecture
     :ivar created_on: Time and date an artifact was created
-    :vartype created_on: :class:`~datetime.datetime`
+    :vartype created_on: datetime.datetime
     :ivar str digest: Digest for the artifact
     :ivar last_updated_on: Time and date an artifact was last updated
-    :vartype last_updated_on: :class:`~datetime.datetime`
-    :ivar str operating_system: Operating system for the artifact
+    :vartype last_updated_on: datetime.datetime
+    :ivar operating_system: Operating system for the artifact
+    :vartype writeable_properties: ~azure.containerregistry.ArtifactOperatingSystem
```

Comment:
It looks like this should be?

```suggestion
    :vartype operating_system: ~azure.containerregistry.ArtifactOperatingSystem
```